components/resources/wallet_strings.grdp

```diff
@@ -454,10 +459,10 @@
   <message name="IDS_BRAVE_WALLET_MISSING_GAS_LIMIT_ERROR" desc="Transaction has missing gas limit information">Missing gas limit</message>
   <message name="IDS_BRAVE_WALLET_ZERO_BALANCE_ERROR" desc="Send amount must be greater than 0 error">Amount must be greater than 0</message>
   <message name="IDS_BRAVE_WALLET_ADDRESS_REQUIRED_ERROR" desc="Address field is required error">To address is required</message>
-  <message name="IDS_BRAVE_WALLET_INVALID_RECIPIENT_ADDRESS" desc="Invalid recipient address error">Invalid recipient address</message>
+  <message name="IDS_BRAVE_WALLET_INVALID_RECIPIENT_ADDRESS" desc="Invalid recipient address error">This Ethereum address looks invalid.</message>
   <message name="IDS_BRAVE_WALLET_INVALID_ZCASH_UNIFIED_RECIPIENT_ADDRESS" desc="Invalid recipient zcash unified address error">Transparent part missing</message>
```

Comment:
`IDS_BRAVE_WALLET_INVALID_RECIPIENT_ADDRESS` `braveWalletInvalidRecipientAddress` was not an ETH-only error message, but now not referenced at all.

Reply:
Good catch! Removed this string since it is no longer needed!